index.js

```diff
@@ -1,127 +1,134 @@
+  yargs.showHelp()
+  process.exit(0)
 }
 
-function findJsFiles(dir) {
-  return new Promise(function (resolve, reject) {
-    var files = [];
-    findit(dir).on('file', function (file) {
-      // only return files ending in .js
-      if (/\.js$/.test(file)) {
-        files.push(file);
+function runCodeshift (transformName, files) {
+  try {
+    const cmd = require.resolve('jscodeshift/bin/jscodeshift.sh')
+    const transform = path.resolve(`${process.cwd()}/${transformName}`)
+    const child = spawn(cmd, ['-c', os.cpus.length, '--parser', argv.p, '-t', transform].concat(files))
+    child.progress((childProcess) => {
+      if (yargs.v) {
+        childProcess.stdout.pipe(process.stdout)
+      } else {
+        childProcess.stdout.on('data', (data) => {
+          if (/^Results: /.test(String(data))) {
+            console.log(String(data).replace(/\n$/, ''))
+          }
+        })
       }
-    }).on('end', function () {
-      resolve(files);
-    }).on('error', reject);
-  });
+      childProcess.stderr.pipe(process.stderr)
+    })
+    return child
+  } catch (error) {
+    console.error(error)
+    return Promise.reject(error)
+  }
 }
 
-function runCodeshift(transformName, files) {
-  var cmd = require.resolve("jscodeshift/bin/jscodeshift.sh");
-  var transform = require.resolve('5to6-codemod/transforms/' + transformName);
-  var child = spawn(cmd, ["--parser", yargs.argv.parser, "-t", transform].concat(files));
-  child.progress(function (childProcess) {
-    if (verbose) {
-      childProcess.stdout.pipe(process.stdout);
-    } else {
-      childProcess.stdout.on('data', function (data) {
-        if (/^Results: /.test(String(data))) {
-          console.log(String(data).replace(/\n$/, ''));
-        }
-      });
-    }
-    childProcess.stderr.pipe(process.stderr);
-  });
-  return child;
-}
+async function codeSwitching (files) {
+  // Require -> Import
+  console.log(`Transforming ${colors.yellow('require()')} to ${colors.cyan('import')} ...`)
+  await runCodeshift('5to6-codemod/transforms/cjs.js', files)
 
-function derequireify(files) {
-  console.log('\nTransforming ' + colors.yellow('require()') + ' to ' +
-    colors.cyan('import') + ' ...');
-  return runCodeshift('cjs.js', files);
-}
+  // module.exports -> Export
+  console.log(`Transforming ${colors.yellow('module.exports')}/${colors.red('exports')} to ${colors.cyan('export')} ...`)
+  await runCodeshift('5to6-codemod/transforms/exports.js', files)
+
+  // Safe function to arrow conversion
+  console.log(`Transforming (safely) ${colors.yellow('function ()')} to ${colors.cyan('() =>')} ...`)
+  await runCodeshift('js-codemod/transforms/arrow-function.js', files)
+
+  // Use const & let
+  console.log(`Transforming ${colors.yellow('var')} to ${colors.red('const')} or ${colors.cyan('let')} ...`)
+  await runCodeshift('js-codemod/transforms/no-vars.js', files)
```

Comment:
It was never my intention to have `cjs-to-es6` do anything other than transforming exports/import syntax. There are arguably lots of transforms you could run to modernize your JS, and I would prefer to keep this package focused on just import/export.

Reply:
Or maybe I'm wrong and we should just keep it since I'm not really interested in maintaining this package anyway. :stuck_out_tongue: Plus given the name, it really sounds like it does more than just import/export.

Reply:
@nolanlawson Yeah, when I first found this, I did think it did more; and I haven't found a good one that does do more (seems like a lot of folks are relying on Babel & packagers to handle code migration).